packages/lesmis/src/views/DashboardView.js

```diff
@@ -9,7 +9,7 @@ import MuiContainer from '@material-ui/core/Container';
 import Typography from '@material-ui/core/Typography';
 import { useUrlParams } from '../utils';
 import { useVitalsData } from '../api';
-import { EntityVitalsItem } from '../components';
+import { EntityVitalsItem, getPartnerSupportItems } from '../components';
```

Comment:
Since we are going to change this import to "PartnerLogo" this comment is just for future reference. It is conventional in react to always return a react component as opposed to a function from a component file.  If you want to return a component which has a list of components it would looks something like: `<PartnerSupportList vitals={vitals} />`